Source/WebCore/platform/graphics/gbm/DMABufReleaseFlag.h

```diff
@@ -78,6 +78,22 @@ struct DMABufReleaseFlag {
             WTFLogAlways("Error writing to the eventfd at DMABufReleaseFlag: %s", safeStrerror(errno).data());
     }
 
+    template<class Encoder> void encode(Encoder& encoder) &&
```

Comment:
I notice the style checker got confused by the rvalue qualifier. I wonder if we have a bug report for that.